docs/src/css/custom.css

```diff
@@ -7,23 +7,64 @@
 /* You can override the default Infima variables here. */
 :root {
   --ifm-font-family-base: "Plain-Regular";
+  --ifm-color-primary: #8C7EFF;
+  --ifm-color-primary-dark: #6e5bff;
+  --ifm-color-primary-darker: #5f4aff;
+  --ifm-color-primary-darkest: #3216ff;
+  --ifm-color-primary-light: #aaa1ff;
+  --ifm-color-primary-lighter: #b9b2ff;
+  --ifm-color-primary-lightest: #e6e3ff;
+  --ifm-code-font-size: 95%;
+  --docusaurus-highlighted-code-line-bg: rgba(0, 0, 0, 0.1);
+  --ifm-link-color: #8C7EFF;
+  --ifm-menu-color-active: #8C7EFF;
+  --ifm-navbar-link-hover-color: #8C7EFF;
 }
 
+.alert--danger {
+  --ifm-alert-background-color: #ffd1cb;
+  --ifm-alert-background-color-highlight: #ffbab1;
+  --ifm-alert-foreground-color: rgb(66,69,73);
+  --ifm-alert-border-color: #ff7764;
+}
+
+.alert--warning {
+  --ifm-alert-background-color: #ffe2b1;
+  --ifm-alert-background-color-highlight: #ffd998;
+  --ifm-alert-foreground-color: rgb(66,69,73);
+  --ifm-alert-border-color: #ffc564;
+}
+
+.alert--info {
```

Comment:
ok i do not know colour theory but i made them darker for you

Reply:
Not colour theory, there was no light/dark mode in the modified css. Which regressed from the existing site.
Looks good now though!

Reply:
haha designing a dark mode is colour theory. it doesn't use aztec colours because we don't have a dark mode media kit yet. so this will probably change but as long as it's not burning people's retinas for now we should be good